spec/board_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'board'
+
+module RubyConnect
+  describe Board do
+    let(:board) { subject }
+
+    context 'when a move is still not done' do
+      it 'is empty' do
+        expect(board).to be_empty
+      end
+    end
+
+    context 'when a move is done' do
```

Comment:
What does it means for a move to 'be done'? I think we could rename this to 'after a player selects a column'.